src/common/inc/texpr.h

```diff
@@ -74,8 +125,19 @@ typedef struct tExprNode {
     } _node;
 
     struct SSchema     *pSchema;
+
     tVariant           *pVal;
+
+    struct {
+      int16_t functionId;
+      int32_t numChildren;
+      struct tExprNode **pChildren;
+    } _func;
+
+    TAOS_FIELD        *pType;
```

Comment:
TAOS_FIELD is defined in taos.h of the client API. Not recommended in internal implementations.